packages/kbn-build/src/production/build_production_projects.integration_test.js

```diff
@@ -1,25 +1,18 @@
-import expect from 'expect.js';
 import tempy from 'tempy';
 import copy from 'cpy';
 import { resolve } from 'path';
 import globby from 'globby';
 
-import { buildProductionProjects } from '../build_production_projects';
-import { getProjects } from '../../utils/projects';
-
-// This is specifically a Mocha test instead of a Jest test because it's slow
-// and more integration-y, as we're trying to not add very slow tests to our
-// Jest unit tests.
+import { buildProductionProjects } from './build_production_projects';
+import { getProjects } from '../utils/projects';
 
 describe('kbn-build production', function() {
   it('builds and copies projects for production', async function() {
-    this.timeout(60 * 1000);
+    jest.setTimeout(60 * 1000);
```

Comment:
Does this change the timeout for all tests of just this specific test? I've usually done:

```
test('some description', () => {
}, 60 * 1000);
```

to set specific timeouts.

(and yep, the Jest docs on this is ... 💩)

Reply:
> Does this change the timeout for all tests of just this specific test?

Hmm, I'll double check that other tests aren't affected

Reply:
Btw, the `timeout` arg is documented in https://facebook.github.io/jest/docs/en/api.html#methods

Reply:
👍

Reply:
Haha, yeah, that was a bit heavy. Shit happens — Prettier won't always be _perfect_.